test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -802,7 +803,8 @@ context('action sidebar', function() {
               created_at: '2019-08-25T14:15:22Z',
             },
             meta: {
-              download: 'https://www.bucket_name.s3.amazonaws.com/patients/1/HRA%20v2.pdf',
+              view: 'https://www.bucket_name.s3.amazonaws.com/patients/1/view/HRA%20v2.pdf',
+              download: 'https://www.bucket_name.s3.amazonaws.com/patients/1/download/HRA%20v2.pdf',
```

Comment:
Don't know what the difference between these two links will be.

But this just creates two different links and tests that they are used in the correct spots. Filename vs. Download button.

Reply:
Technically the path won't be different, but rather the query string will be modified. NBD.

Reply:
Ah gotcha. We can update this test later to be more accurate compared to qa/prod.

Reply:
meh it's fine, these strings could be `view-url-goes-here` and it's enough for these integration tests.

Reply:
Yup, exactly. The important part is just differentiating between download/upload/view.

Reply:
Works for me!